### ISSUE TITLE
silence colormap warnings from Reader

### ISSUE THREAD

**Issue by ageller** [ageller/Firefly#145]
Let's stop this from printing every time the Reader is imported (and therefore every time the server is started). 

```
>>> from firefly.data_reader.reader import Reader
palettable colormaps are not installed
don't have phil's colormaps
```

**Comment by agurvich** [comment]
issue is with my abg_python, we need to update the version required in setup.py. let me check which version of abg_python has those print messages suppressed

**Comment by agurvich** [comment]
closed by 70a715f8a825151f2bee950162ffb9177ca98bbd
